src/routes/overviews/bufnmf.svx

```diff
@@ -0,0 +1,134 @@
+---
+title: Audio Decomposition using BufNMF
+blurb: |
+    Decompose an audio buffer into component parts using non-negative matrix factorization.
+tags:
+    - decomposition
+flair: article
+---
+
+<script>
+    import Image from '$lib/components/Image.svelte';
+    import Audio from '$lib/components/Audio.svelte';
+</script>
+
+# Decomposition
+
+"NMF" stands for Non-negative Matrix Factorization, which is a machine learning algorithm that can be used for decomposing an audio buffer into component parts. In this example, we’ll look at decomposing a mono drum loop into three different mono audio buffers, each hopefully containing the hits from just one of the instruments: kick, snare, and cymbal. Here is the recording and spectrogram of our drum loop.
+
+<Audio
+src="/audio/Nicol-LoopE-M.mp3"
```

Comment:
I'll need to check this but we should compress the life out of this till its as small as possible. My usual settings are 128kbps with the `ffmpeg` or `REAPER` converter. For comparison the oboe sample is 5 times as long but about the same size currently. Smaller is better for making the page __fast__.